pytorch_lightning/utilities/distributed.py

```diff
@@ -203,10 +204,18 @@ def all_gather_ddp_if_available(
     Return:
         A tensor of shape (world_size, batch, ...)
     """
+    if group is None:
+        group = globals()["group"].WORLD
```

Comment:
Why do we do this over just `group.WORLD`?

Reply:
removed